test-jest/specs/chart/PieChart.spec.tsx

```diff
@@ -131,18 +123,16 @@ describe.only('<PieChart />', () => {
         />
       </PieChart>,
     );
-    const sectors = wrapper.find(Sector);
-    const se = sectors.at(2);
+    const sectors = container.querySelectorAll('.recharts-sector');
+    const se = sectors[2];
 
-    se.simulate('click');
-    expect(onClick.calledOnce).to.equal(true);
+    fireEvent.click(se);
+    expect(onClick).toBeCalled();
 
-    /*
-    se.simulate('mouseEnter');
-    expect(onMouseEnter.calledOnce).to.equal(true);
+    fireEvent.mouseEnter(se);
+    expect(onMouseEnter).toBeCalled();
 
-    se.simulate('mouseLeave');
-    expect(onMouseLeave.calledOnce).to.equal(true);
-    */
+    fireEvent.mouseLeave(se);
+    expect(onMouseLeave).toBeCalled();
```

Comment:
Either change the name of the test to reflect the scope, or preferably, split this up into two separate tests

`test('click on Sector should invoke onClick callback')`

and 

`test('Mouse enter and mouse leave are triggered'`

and possibly even another for Mouse over and Mouse out?

Reply:
Agreed, I was more attempting to get things working as they were before adding net new tests/changing test names (unless it made sense to do so). Since this PR is small I'll make the change, but this might be harder to do until initial changes are made in larger test files like `LineChart`